Source/RK3/RK3.H

```diff
@@ -69,8 +73,8 @@ void CalcDiffFlux(const MultiFab& cons_in,
                   const amrex::Geometry geom,
                   const amrex::Real* dx, const amrex::Real dt);
 
-void conservedToPrimitive(MultiFab& prim_in, const MultiFab& cons_in);
+//void conservedToPrimitive(MultiFab& prim_in, const MultiFab& cons_in);
 
-void primitiveToConserved(const MultiFab& prim, MultiFab& cons);
+//void primitiveToConserved(const MultiFab& prim, MultiFab& cons);
 
```

Comment:
Are these remnants from PeleC? Looks like these are never defined.

Reply:
Yeah, that's why I commented it out. Should we just get rid of them?

Reply:
I think so